networks/rpc/service.go

```diff
@@ -118,19 +120,25 @@ func (r *serviceRegistry) callback(method string) *callback {
 		namespace = "kaia"
 	}
 
-	return r.services[namespace].callbacks[method]
+	cb := r.services[namespace].callbacks[method]
+	logger.Trace("lookup callback", "method", reqMethod, "ok", cb != nil)
+	return cb
```

Comment:
@hyeonLewis No need to bring in logging in this file. I added it for debugging, but it's too wordy here.

Reply:
Ok, I thought it needs to be identical. Then can I also apply cleanup like below?
```go
func (s *Server) RegisterName(name string, rcvr interface{}) error {
	if name == "klay" || name == "kaia" {
		if err := s.services.registerName("klay", rcvr); err != nil {
			return err
		}
		if err := s.services.registerName("kaia", rcvr); err != nil {
			return err
		}
		return nil
    } 
    return s.services.registerName(name, rcvr)
}
```

Reply:
Ok, I thought it needs to be identical. Thanks.

Reply:
Applied and force-pushed.